src/blsct/aggregationsession.cpp

```diff
@@ -951,6 +951,7 @@ void AggregationSessionThread()
                 MilliSleep(1000);
             } while (true);
 
+            MilliSleep(5000);
```

Comment:
should we maybe let the user specify the timeouts so they can try with different settings depending on their specs?